### PR TITLE
Test bunny file

### PR DESCRIPTION
**Pull request by mQkatz**
This PR adds the test file "bunny.js" to our repo.

### CODE REVIEW

File: bunny.js [hunk 1]
```diff
@@ -0,0 +1 @@
+bunnymen
```
Comment (samueladam007) [inline bunny.js:1]: LGTM!

**Review by samueladam007 — COMMENTED** [review]
LGTM

**Comment by samueladam007** [comment]
LGTM

**Comment by mQkatz** [comment]
Closing this pull request as our shopping list shall not include bunnies.

**Pull request closed without merging**